ccmlib/common.py

```diff
@@ -185,19 +185,11 @@ def make_cassandra_env(install_dir, node_path, update_conf=True):
     if not os.path.exists(dst):
         shutil.copy(orig, dst)
 
-    if update_conf:
-        replacements = ""
-        if is_win() and get_version_from_build(node_path=node_path) >= '2.1':
-            replacements = [
-                ('env:CASSANDRA_HOME =', '        $env:CASSANDRA_HOME="%s"' % install_dir),
-                ('env:CASSANDRA_CONF =', '    $env:CCM_DIR="' + node_path + '\\conf"\n    $env:CASSANDRA_CONF="$env:CCM_DIR"'),
-                ('cp = ".*?env:CASSANDRA_HOME.conf', '    $cp = """$env:CASSANDRA_CONF"""')
-            ]
-        else:
-            replacements = [
-                ('CASSANDRA_HOME=', '\tCASSANDRA_HOME=%s' % install_dir),
-                ('CASSANDRA_CONF=', '\tCASSANDRA_CONF=%s' % os.path.join(node_path, 'conf'))
-            ]
+    if update_conf and not (is_win() and get_version_from_build(node_path=node_path) >= '2.1'):
```

Comment:
This could do with some documenting comments and/or pulling out the guard logic into a variable.